osu.Game/Overlays/Settings/Sections/Graphics/LayoutSettings.cs

```diff
@@ -219,6 +230,13 @@ void updateScalingModeVisibility()
                 scalingSettings.AutoSizeAxes = scalingMode.Value != ScalingMode.Off ? Axes.Y : Axes.None;
                 scalingSettings.ForEach(s =>
                 {
+                    if (s == dimSlider)
+                    {
+                        s.TransferValueOnCommit = false;
+                        s.CanBeShown.Value = scalingMode.Value == ScalingMode.Everything || scalingMode.Value == ScalingMode.ExcludeOverlays;
```

Comment:
This is not going to work well. See `AutoSizeAxes` above which is doing other wooaefjjjjj

Reply:
I think this is okay in hindsight. It's just a very convoluted update method in the first place. I've refactored slightly to a point I'm happy with it.